e2e/.circleci/config.yml

```diff
@@ -23,6 +23,9 @@ jobs:
           command: |
             set +e
             npm install
+            mkdir bak
+            cp ./node_modules/chromedriver/lib/chromedriver/chromedriver bak/
```

Comment:
Were these two lines remnants from debugging? Seems that if the replacement from `/usr/local/bin/chromedriver` works, there is no need to store the old file.

Reply:
Thanks for catching this.  I've removed these two lines.